_episodes/44.md

```diff
@@ -0,0 +1,213 @@
+has grown to be a well known and widely used BI application.
+
+A Presto driver was created in 2018. It directly integrated with the client REST
+API. With the rename of Presto to Trino, Manfred [created a
+PR](https://github.com/metabase/metabase/pull/15160) that replicates this for
+Trino to ensure continued support for the community. In the discussion it was
+decided that it would be better to use the Trino JDBC driver, similar to how
+other drivers for Metabase work.
+
+After some more demand from the user and customer community, Starburst and
+Metabase established a collaboration, and started implementation of the current
+driver. Piotr led the charge, Andrew buckled down and learned Clojure, and
+together a first release was created and tested. The driver is now provided as
+an open source project managed by Starburst.
+
+## Core advantages of using Metabase with Trino
+
+With Metabase and the driver for Trino, Trino users have access to a well
+established and proven open source BI tool. It is suitable for internal usage in
+any organization, and users can upgrade to commercial version for more demanding
+deployments and use cases.
+
+The combination of Trino and Metabase also provides a number of unique benefits
+for Metabase users that are not available with typical drivers for systems.
+These are typically databases that support SQL, and are limited to the specific
+database.
+
+With Trino and the driver, you have access to the following unique features:
+
+* Metabase users can connect to databases that do no yet have a Metabase driver,
+  but are supported by Trino
+* Trino also enables using SQL for system that don't support SQL such as MongoDB
+  or Elasticsearch, and therefore allows Metabase usage with these systems.
+* With Trino you can join data from different catalogs in the same SQL query.
+  This also applies to Metabases  reports or visualizations.
+
+Metabase also includes support for access control for any connected datasource,
+all the way to row level security. This includes Trino and can be used to secure
+Trino access through Metabase to a large group of your Trino users, such as all
+BI users.
+
+## Demo of the episode: Metabase and Trino
+
+Luis shows us the demo from his repository at
+[https://github.com/paoliniluis/metabase-trino](https://github.com/paoliniluis/metabase-trino).
+Watch our video to see it and action, and check out the instructions in the
+repository to try yourself.
+
+## Real world use cases at Meesho
+
+<img src="../assets/images/logos/meesho-small.png"/>
+
+[Meesho](https://www.meesho.com/) is India’s fastest growing internet commerce
+company. They provide a large retail website and support small business
+entrepreneurs with their platform.
+
+Meesho relies on the Trino, Metabase and the Trino Metabase driver from
+Starburst for their data platform.
+
+Piotr and Luis share more details about the celebrated successes.
+
+## PR of the episode
```

Comment:
@andrewdibiasio6 this can be mostly yours to talk about whatever you like .. the rest of us can chime in with questions.

Reply:
The only thing I will cover is my experience working with metabase and I will go over the README here https://github.com/starburstdata/metabase-driver and show people how to use the commands to build/run/test metabase + the metabase starburst driver. Likely won't need too much time.

Reply:
Perfect .. thats great.